src/components/screens/management-screen.tsx

```diff
@@ -0,0 +1,438 @@
+        </Box>
+        <ResponsiveContainer className={ classes.pieChartContainer }>
+          <PieChart>
+            <Pie
+              cx="50%"
+              cy="50%"
+              dataKey="total"
+              data={ workTimeData }
+              label={ renderCustomizedLabel }
+            >
+              { workTimeData.map((entry, index) => (
+                <Cell fill={ COLORS[index % COLORS.length] }/>
+              )) }
+            </Pie>
+            <RechartTooltip content={ renderCustomizedTooltip }/>
+          </PieChart>
+        </ResponsiveContainer>
+        <Box className={ classes.personRedirect }>
+          <Divider/>
+          <Button
+            onClick={ () => handlePersonRedirectClick(person) }
+            className={ classes.personRedirectButton }
+          >
+            <Typography style={{ fontWeight: 600 }}>
+              { strings.managementScreen.seeMore }
+            </Typography>
+            <KeyboardArrowRightIcon/>
+          </Button>
+        </Box>
+      </Paper>
+    );
+  };
+
+  /**
+   * Renders person entry
+   * 
+   * @param personTotalTime person total time data
+   */
+  const renderPersonEntry = (personTotalTime: PersonWithTotalTime) => {
+    const { person, timeEntryTotal } = personTotalTime;
+
+    if (!person.active || !timeEntryTotal || timeEntryTotal.expected === 0) {
+      return;
+    }
+
+    return (
+      <ListItem
+        button
+        disableRipple
+        disableTouchRipple
+        selected={ selectedPersonWithTotalTime?.person.id === person.id }
+        onClick={ event => handleListItemClick(event, personTotalTime) }
+        className={ classes.personListEntry }
+      >
+        <Paper className={ classes.personEntry }>
+          <Box className={ classes.userInfoContainer }>
+            <Typography variant="h2">
+              { `${person.firstName} ${person.lastName}` }
+            </Typography>
+            <Typography
+              variant="h4"
+              className={ classes.personEntryDate }
+            >
+              { `${moment(person.startDate).format("DD.MM.YYYY")}-` }
+            </Typography>
+          </Box>
+          <Box className={ classes.personEntrySubtitle } >
+            <Tooltip
+              title={ `${strings.expected}: ${TimeUtils.convertToHours(timeEntryTotal!.expected)}, ${strings.logged}: ${TimeUtils.convertToHours(timeEntryTotal!.logged)}` }
+            >
+              <Typography
+                className={ classes.personEntryTime }
+                style={{
+                  color: timeEntryTotal!.total >= 0 ? theme.palette.success.main : theme.palette.error.main
+                }}
+              >
+                { TimeUtils.convertToHours(timeEntryTotal!.total) }
+              </Typography>
+            </Tooltip>
+          </Box>
+        </Paper>
+      </ListItem>
+    );
+  };
+
+  /**
+   * Renders bottom padding 
+   */
+  const renderBottomPadding = () => {
```

Comment:
Simple renders like this can use the () => (); syntax instead of () => { return ... };